acceptance-tests/profiler-stress/runner.cs

```diff
@@ -174,6 +179,9 @@ static int Main ()
 				info.EnvironmentVariables.Clear ();
 				info.EnvironmentVariables.Add ("MONO_PATH", classDir);
 
+				if (suspend)
```

Comment:
Why not just having the check for `Environment.GetEnvironmentVariable ("MONO_PROFILER_STRESS_SUSPEND")` here?

Reply:
No need to `getenv` on every iteration.